index.js

```diff
@@ -18,67 +18,69 @@ module.exports = function(bot){
 
   let scheduler = () => bot.Scheduler || Scheduler;
 
-  var room = process.env.HUBOT_JOB_CHANNEL;
+  var mainRoom = process.env.HUBOT_JOB_CHANNEL;
 
-  function sendMessage(bot, message, msg) {
+  function sendMessage(bot, message, msg, room = mainRoom) {
     bot.messageRoom(room, msg);
-    if (message && message.envelope && message.envelope.room !== room){
+    if (message && message.envelope && message.envelope.room !== mainRoom) {
```

Comment:
Why is this changed to mainRoom?  Don't you still want to be checking just room?

Reply:
oh yeah - I changed the name of the default `room` variable from `process.env.HUBOT_JOB_CHANNEL;` to `mainRoom` to avoid confusion since a couple of the parameters are now called room. Maybe it makes sense to change the room parameter to `channel` and maintain the variable`room`.